components/docs-chef-io/content/automate/ha_backup_restore_object_storage.md

```diff
@@ -40,7 +47,19 @@ This section provides the pre-backup configuration required to back up the data
 - `chown -RL hab:hab /hab/svc/automate-ha-opensearch/config/opensearch.keystore` (Setting hab:hab permission)
 - `curl -k -X POST "https://127.0.0.1:9200/_nodes/reload_secure_settings?pretty" -u admin:admin` (Command to load the above setting)
 
-The final output after running the curl command on all nodes is given below:
+#### For _GCS_
+This section provides the pre-backup configuration required to back up the data on GCP Object Storage. The steps to set a secret key using commands are given below:
+
+1. Log in to **all** the OpenSearch nodes and follow the steps on all the OpenSearch nodes.
+
+- `export OPENSEARCH_PATH_CONF="/hab/svc/automate-ha-opensearch/config"`
+- `export GCS_SERVICE_ACCOUNT_JSON_FILE_PATH="/path/to/googleServiceAccount.json"` (Provide the file path of your googleServiceAccount.json file. Note that the content of this json file should looks like the snippet depicted [here](#structure-of-google-service-account-json-file))
+- `chown -RL hab:hab $GCS_SERVICE_ACCOUNT_JSON_FILE_PATH`
+- `hab pkg exec "$OS_ORIGIN_NAME/$OS_PKG_NAME" opensearch-keystore add-file --force gcs.client.default.credentials_file $GCS_SERVICE_ACCOUNT_JSON_FILE_PATH`
+- `chown -RL hab:hab /hab/svc/automate-ha-opensearch/config/opensearch.keystore` (Setting hab:hab permission)
+- `curl -k -X POST "https://127.0.0.1:9200/_nodes/reload_secure_settings?pretty" -u admin:admin` (Command to load the above setting)
```

Comment:
why to use password in the curl 
curl  --cacert /hab/svc/automate-ha-opensearch/config/certificates/root-ca.pem --key /hab/svc/automate-ha-opensearch/config/certificates/admin-key.pem --cert /hab/svc/automate-ha-opensearch/config/certificates/admin.pem https://localhost:9200/_cat/indices

Reply:
Done